lectures/index.md

```diff
@@ -86,7 +86,23 @@ Note not all of the provided examples were covered in the lecture but the are al
 ### Week 2 - The DOM and Events
 <hr class="underline" style="background: var(--info)">
 
-<div class="blockheading" style="border-color: var(--info)">Tuesday Lecture: TBA</div>
+<div class="blockheading" style="border-color: var(--info)">Tuesday Lecture: DOM API and Events</div>
+- [Download Slides](lectures/pdfs/lecture-3-dom-and-events.pdf)
+
+- Scrolling Example
+    - [Live](lectures/lecture3/scrolling)
+    - [HTML](render.html?l=html&c=lectures/lecture3/scrolling/scrolling.html)
+    - [JS](render.html?l=js&c=lectures/lecture3/scrolling/scrolling.js)
+
+- Keyboard Events Example
+    - [Live](lectures/lecture3/keyboard)
```

Comment:
this only works if the folder contains a index.html, if it doesn't you'll have to do lectures/lecture3/scrolling/scrolling.html